R/z_animintHelpers.R

```diff
@@ -173,13 +173,30 @@ hjust2anchor <- function(hjust){
   }
 }
 
+#' @param params Named list of layer parameters
+#' @return NULL if valid, stops with error if invalid showSelected variant found
+validateShowSelectedParams <- function(params) {
+  if (is.null(params) || length(params) == 0 || is.null(names(params))) {
```

Comment:
these conditions are redundant, we only need is.null(names(params)) (not the others)